### DISCUSSION TITLE
Ajuda

### DISCUSSION THREAD

**Discussion by Gerusinha** [aclap-dev/video-downloadhelper#392] [License & Purchase requests]
### What's your issue?

Comprei a licença. A chave foi ativada automaticamente e a extensão Video DownloadHelper não está funcionando.
[Teste.pdf](https://github.com/aclap-dev/video-downloadhelper/files/14925263/Teste.pdf)
[Teste.pdf](https://github.com/aclap-dev/video-downloadhelper/files/14925267/Teste.pdf)

### The 10 first digits (only 10) of your license number or email address

gerusamsg@

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Do you have a public link to test?

**Reply by Mattmjs** [reply]
It's a google drive problem like the other post: https://github.com/aclap-dev/video-downloadhelper/discussions/383
No media detected from google drive.

**Comment by Gerusinha** [comment]
jcfrog -  link público pegou o privado não. O que fazer? obrigado

**Reply by jcfrog** [reply]
as @Mattmjs said, it seems it's not detected yet, I'm afraid there is no solution for the moment. see https://github.com/aclap-dev/video-downloadhelper/discussions/383
